.travis.yml

```diff
@@ -0,0 +1,10 @@
+language: c
```

Comment:
ocaml isn't supported by travis, which is why we're using bash files as testing scripts and the weird travis config process